src/mame/layout/enmirage.lay

```diff
@@ -0,0 +1,210 @@
+-->
+
+<mamelayout version="2">
+
+	<!-- Backgrounds -->
+	<element name="background"><rect><color red="0.243" green="0.203" blue="0.164" /></rect></element>
+	<element name="lcd_background"><rect><color red="0.360" green="0.274" blue="0.180" /></rect></element>
+
+	<!-- Labels -->
+	<element name="label_1"> <text string="1"></text></element>
+	<element name="label_2"> <text string="2"></text></element>
+	<element name="label_3"> <text string="3"></text></element>
+	<element name="label_Rec"> <text string="Rec"></text></element>
+	<element name="label_Upper"> <text string="Upper"></text></element>
+	<element name="label_4"> <text string="4"></text></element>
+	<element name="label_5"> <text string="5"></text></element>
+	<element name="label_6"> <text string="6"></text></element>
+	<element name="label_Play"> <text string="Play"></text></element>
+	<element name="label_Lower"> <text string="Lower"></text></element>
+	<element name="label_Param"> <text string="Param"></text></element>
+	<element name="label_Value"> <text string="Value"></text></element>
+	<element name="label_7"> <text string="7"></text></element>
+	<element name="label_8"> <text string="8"></text></element>
+	<element name="label_9"> <text string="9"></text></element>
+	<element name="label_Load"> <text string="Load"></text></element>
+	<element name="label_Off"> <text string="Off/&#9662;"></text></element>
+	<element name="label_On"> <text string="On/&#9650;"></text></element>
+	<element name="label_Cancel"> <text string="Cancel"></text></element>
+	<element name="label_0"> <text string="0/Prog"></text></element>
+	<element name="label_Enter"> <text string="Enter"></text></element>
+	<element name="label_Save"> <text string="Save"></text></element>
+	<element name="label_CONTROL"> <text string="CONTROL"></text></element>
+	<element name="label_SELECT"> <text string="SELECT"></text></element>
+	<element name="label_SEQ"> <text string="SEQ"></text></element>
+	<element name="label_SAMPLE"> <text string="SAMPLE"></text></element>
+	<element name="label_LOAD"> <text string="LOAD"></text></element>
+
+	<!-- Buttons -->
+	<element name="button_yellow"><rect><color red="0.996" green="0.874" blue="0.016" /></rect></element>
+	<element name="button_grey"><rect><color red="0.5" green="0.5" blue="0.5" /></rect></element>
+	<element name="button_black"><rect><color red="0" green="0" blue="0" /></rect></element>
+	<element name="button_white"><rect><color red="0.978" green="0.888" blue="0.825" /></rect></element>
+
+	<element name="led_segement" defstate="0">
+		<rect state="0"><color red="0.85" green="0.87" blue="0.36" alpha="0.1" /></rect>
+		<rect state="1"><color red="0.85" green="0.87" blue="0.36" /></rect>
+	</element>
+
+	<element name="led_indicator" defstate="0">
+		<disk state="0"><color red="0.85" green="0.87" blue="0.36" alpha="0.1" /></disk>
+		<disk state="1"><color red="0.85" green="0.87" blue="0.36" /></disk>
+	</element>
+
+	<view name="Default Layout">
```

Comment:
There still seem to be a large number of unnecessary `name` attributes on elements in this view.  Are the labels really interactive in some way?